digitalocean_plugin/tests/test_security.py

```diff
@@ -0,0 +1,278 @@
+########
```

Comment:
i will if you make me. 

but i'd argue that your test code is too complicated if you need to do such a thing.

do you make me? ;)